### PR TITLE
helm create: make generated YAML indentation more consistent

### PR DESCRIPTION
**Pull request by consideRatio**
**What this PR does / why we need it**:
It just makes the indentation of generated output of `helm create` indented more consistently.

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]
needs another maintainer to approve before merging.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks for this @consideRatio

**Comment by bacongobbler** [comment]
As a note, commits do not need to be cherry-picked into a minor release. As per the release checklist, the next minor release is cut from the HEAD of the master branch. Cherry picks are only necessary for patch releases.

https://helm.sh/docs/community/release_checklist/#1-create-the-release-branch

Removing the needs-pick label accordingly. :)

**Pull request merged**
